source/nodejs/adaptivecards-designer/src/tool-box.ts

```diff
@@ -130,6 +131,7 @@ export class Toolbox {
         this._expandCollapseButtonElement.tabIndex = 0;
         this._expandCollapseButtonElement.setAttribute("role", "button");
         this._expandCollapseButtonElement.setAttribute("aria-expanded", "true");
+        this._expandCollapseButtonElement.setAttribute("aria-labelledby", this.title.replace(/ /g, "-"));
```

Comment:
Can you define a new label for it, instead of deducing it from title.

Reply:
Yes I can do that instead